server/routes/admin.js

```diff
@@ -1,12 +1,14 @@
 import express from 'express';
+import middlewares from '../middlewares';
 import AuthorValidator from '../middlewares/AuthorValidator';
```

Comment:
You are importing from the middlewares file twice. You should import ones and destructure

Reply:
Thank you for the observation, I have destructured it.